sample_example/README.md

```diff
@@ -1,12 +1,19 @@
 # Toy example
 
 Here we present a walkthrough of running `make_prg` on a toy example.
-Note that there is no need to have `make_prg` installed.
-The test script downloads the precompiled binary of the latest release.
 We run:
 1) `make_prg from_msa` to create PRGs from MSAs;
 2) `make_prg update` to update PRGs with denovo paths;
 
+# Dependencies
+
+* `mafft` in `PATH`. Can be installed:
```

Comment:
We have a `--mafft` paramter right? So it doesn't have to be on `$PATH`

Reply:
It has because in this readme we tell the user to just run `./run_make_prg_on_sample_example.sh` to test make_prg on the sample example. But yeah, user can also simply open this script and add the `-mafft` param to `make_prg update`.